crates/symbolicator/src/services/cacher.rs

```diff
@@ -212,6 +213,7 @@ pub trait CacheItemRequest: 'static + Send + Sync + Clone {
         status: CacheStatus,
         data: ByteView<'static>,
         path: CachePath,
+        expiration: ExpirationTime,
```

Comment:
What is the intended use of this parameter?

Reply:
Right now this is unused, but eventually I want to hold these things in an in-memory cache, and I would need a way to figure out if the in-memory cache is valid.

For that, we would also need to convert the `Duration` to an `Instant` at some point.